metaflow/plugins/argo/argo_workflows.py

```diff
@@ -3445,6 +3461,26 @@ def pvc_volumes(self, pvcs=None):
             )
         return self
 
+    def ephemeral_volume_claims(self, claims=None):
```

Comment:
also, are there any use cases when the mode is not `ReadWriteOnce`? if there are, it would be good to document those scenarios and call out the behavior.

Reply:
I think this is fair. I've set the default to be `ReadWriteOnce`. I don't _think_ you'd ever need it to change, but I've left the ability to override just in case.